tests/test_backend/test_user_api.py

```diff
@@ -1,57 +1,29 @@
 import allure
 import pytest
-from addict import Dict
-from core.testlib.utils import get_random_str
 from steps.api_steps.user_api_steps import user_api_steps, user_api_assert_steps
 
 
 @allure.description('RGA-1')
 @allure.title("Register customer via API")
 @allure.tag('Account API')
-@pytest.mark.parametrize('scenario', [
-    Dict(customer_id=12434, repeatedPassword='password',
-         customer=(Dict(firstName=get_random_str(), lastName='test', address=Dict(
-             street='1 Rose Street', city='New York City', state='New York',
-             zipCode='11111'), phoneNumber='987654321', ssn='123456789',
-                        username=get_random_str(), password='password')))])
-def test_register_user(scenario):
-    user_api_steps.clean_db()
-    user_api_steps.initialize_db()
-    user_api_steps.register_user(scenario)
-    user_api_steps.get_user_info(scenario.customer_id)
-    user_api_assert_steps.check_user_data(12434, scenario.customer)
+def test_register_user(user_scenario, clear_initialize_db):
+    user_api_steps.register_user(user_scenario.user_data)
+    user_api_steps.get_user_info(user_scenario.customer_id)
+    user_api_assert_steps.check_user_data(user_scenario.customer_id, user_scenario.user_data.customer)
 
 
 @allure.description('RGA-2')
 @allure.title("Edit User's data via API")
 @allure.tag('Account API')
-@pytest.mark.parametrize('scenario',
-                         [Dict(customer_id=12323, user_data=Dict(firstName='Cindy', lastName='test', address=Dict(
```

Comment:
Constant of the static user should be stored in a constant (or, better to get it from API, for example 
curl -X GET --header "Accept: application/json" "http://localhost:8080/parabank/services/bank/login/${username}/${password}")